restapi/src/test/java/io/stargate/db/datastore/ValidatingDataStore.java

```diff
@@ -223,9 +210,11 @@ public B bindValues(List<TypedValue> values) {
       ExpectedExecution exec = new ExpectedExecution(this, expectation);
       expectedExecutions.add(exec);
       bound = true;
-      this.source = new BoundQuery.Source<>(query, values);
-      //noinspection unchecked
-      return (B) exec;
+      return exec;
+    }
+
+    public boolean hasId(MD5Digest preparedId) {
```

Comment:
where this method is called? I don't see any invocation.

Reply:
good catch, thx! It is not used... leftover from a previous iteration. will remove.

Reply:
removed in the refactored code (force-pushed)